osbs/api.py

```diff
@@ -502,6 +519,27 @@ def create_build(self, **kwargs):
         return self.create_prod_build(**kwargs)
 
     @osbsapi
+    def create_worker_build(self, *args, **kwargs):
+        """
+        Create a worker build
+
+        Pass through method to create_prod_build with the following
+        modifications:
+            - platform param is required
+            - inner template set to worker_inner.json if not set
+            - outer template set to worker.json if not set
+
+        :return: BuildResponse instance
+        """
+        if not kwargs.get('platform'):
```

Comment:
No.

The 'architecture' keyword is a configuration setting to override the implicit value (taken from asking the running system, like 'uname -m') for the 'architecture' label in the image. It is nothing to do with controlling which node should build the image.

IMHO we should deprecate 'architecture' altogether in fact.